source/CardGames.py

```diff
@@ -3,7 +3,7 @@
 import os
 
 cardImages = []
-values = list(range(1,14))
+values = [11, 2, 3, 4, 5, 6, 7, 8, 9, 10, 10, 10, 10]
```

Comment:
This works well. Considering that the 11 will be changed to 1 in the calculateHand() function